config/manager/kustomization.yaml

```diff
@@ -13,6 +13,7 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/netobserv/network-observability-operator
-  newTag: 0.1.3
+  newTag: main
 commonLabels:
   app: network-observability-operator
+  version: main
```

Comment:
can you revert this file change ? I don't know why t added "version: main" on labels, it shouldn't. Anyway, that's a generated file.

Reply:
I know... that's probably because I used a more recent version of the generatior. I'll downgrade to the previous version.

Reply:
or i can upgrade mine, if this is due to operator-sdk version